pkg/k8s/dto.go

```diff
@@ -17,6 +21,37 @@ func (metaDto *MetaDataDto) ObjectMetaFromMetadataDto() metav1.ObjectMeta {
 	}
 }
 
+func (dto *MetaDataDto) Validate() *restErrors.RestErr {
+	newValidator := validator.New()
+	err := newValidator.RegisterValidation("regexp", func(fl validator.FieldLevel) bool {
+		re := regexp.MustCompile("^([a-z]|[0-9])+([a-z]|[0-9]|-)+$")
```

Comment:
name **MUST** start and end with alphabet.
According to this RegEx, it can end with `-`.
Please update.
check issue description #32

Reply:
No it can't,
however i noticed in the response message you wrote the name should end with aphanumeric char as well, so i am changing it to ^([a-z]|[0-9])+([a-z]|[0-9]|-)+([a-z]|[0-9])+$